plugins/common/tls/config_test.go

```diff
@@ -15,10 +15,11 @@ var pki = testutil.NewPKI("../../../testutil/pki")
 
 func TestClientConfig(t *testing.T) {
 	tests := []struct {
-		name   string
-		client tls.ClientConfig
-		expNil bool
-		expErr bool
+		name       string
+		client     tls.ClientConfig
+		expNil     bool
+		expErr     bool
+		serverName string
```

Comment:
`serverName` doesn't seem to be used in this test. Can you remove it please?

Reply:
Yup, vestige of a previous idea for a test - Removing it now.